content/password-cracking-and-how-to-protect-against-them/index.md

```diff
@@ -0,0 +1,187 @@
+
+### Pen Testing 🖋️💻
+**Penetration testing (pen testing)** is a security practice where ethical hackers (cybersecurity experts) attempt to hack into a system (with permission!!!) to assess its security and check where it might be vulnerable, before hackers find the vulnerabilities first. 
+
+Pen testers are hired to check for password vulnerabilities where the testers will employ the same tactics hackers would. They would then report to the company and provide them with a detailed report of their findings. 
+
+### Consistent Training 👩‍🏫
+
+Even with all the tools and techniques we explored, at the end of the day the weakest link is people. **Training** employees and users on the importance of password hygiene is vital to prevent password cracking. Something as simple as leaving your laptop unattended can be a security risk. Which is why some companies require users to log out of their computers before leaving their desks.
+
+## How SuperTokens Protects Against Password Cracking
+
+Phew! That was a lot. Let’s take a closer look at the tools and features SuperTokens provides to enhance account security: 
+
+### Brute Force Attack Detection
+Tracks repeated actions like login attempts or password resets. If too many attempts happen in a short time, it blocks them to protect accounts from attackers.
+
+### Password Breach Detection
+Checks passwords against a database of leaked passwords to see if they've been leaked before. This helps keep accounts safe by avoiding weak passwords.
+
+### New Device Detection
+Recognizes when a user logs in from a new and previously unseen device. This helps find possible unauthorized logins.
+
+### Device Count Tracking
+Monitors the number of unique devices associated with a user account. This helps spot unusual account use. 
+
+### Requester Detection
+Identifies and remembers devices and requester details, even if they try to hide. This helps track the same device across multiple login attempts for better security.
+
+### Multi-Factor Authentication (MFA)
+SuperTokens integrates with MFA to provide an additional layer of security. By requiring multiple forms of verification, such as email or SMS-based one-time passwords (OTPs), MFA ensures that even if a password is compromised, unauthorized access is still prevented. This approach strengthens account protection and makes it significantly harder for attackers to breach accounts. 💪
+
+### Session Management
+SuperTokens securely manages session tokens to protect against threats like session hijacking and fixation. By implementing measures such as access token blacklisting, anti-csrf (cross-site request forgery), token rotation, and cookie consent, it ensures that user sessions remain secure. This prevents attackers from stealing or reusing tokens to gain unauthorized access. 🔒
+
+## Best Practices for Password Security
```

Comment:
hmm, this just rehashes the same content as before right? we could just mention in the previous section that there are best practices also